_episodes/02-accessing.md

```diff
@@ -141,12 +141,18 @@ We can use GitHub to find packages or to make our own packages publicly availabl
 Contrary to CRAN, packages in GitHub are not reviewed.
 This has an advantage: you can have your package published there immediately.
 
-Let's imagine we want to install a package from GitHub, such as [github.com/PabRod/kinematics](https://github.com/PabRod/kinematics).
-The package we installed before, `devtools`, allows us to install easily using:
 
-```r
-devtools::install_github("PabRod/kinematics")
-```
+> ## Find a package on github, bitbucket, or gitlab and install it with devtools.
+> If you have trouble finding a package, try [github.com/PabRod/kinematics](https://github.com/PabRod/kinematics).
+> If you know how to install from github already, try with a package from gitlab. For instance, try [psyverse](https://gitlab.com/r-packages/psyverse)
+>
+> > ## Solution
+> > ```
+    devtools::install_github("PabRod/kinematics")
+    devtools::install_git("https://gitlab.com/r-packages/psyverse")
+    ```
+> {: .solution}
+{: .challenge}
 
 Learning how to publish your package on GitHub is out of the scope of the present course.
 But, if you are interested, we encourage you to take the course on [Version control with Git and GitHub](https://swcarpentry.github.io/git-novice/).
```

Comment:
Maybe this is the point where we should mention git/github integration with Rstudio (see #73)?

Reply:
good idea, fixed!